webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -9731,9 +9746,16 @@ var IS_SOCKET_CONNECTED = false;
                                             if (result && result.data && result.data.response && result.data.response.length !== 0) {
                                                 var updatedAssignee = result.data.response[0];
                                                 mckMessageService.updateAssigneeDetails(updatedAssignee);
+                                                if (updatedAssignee.roleType != 1) {
```

Comment:
@hardikparashar Use this **KommunicateConstants.APPLOZIC_USER_ROLE_TYPE.BOT** instead of using hardcoded value.
For better understanding check all value for object APPLOZIC_USER_ROLE_TYPE